localstack/services/cloudformation/models/ec2.py

```diff
@@ -546,22 +558,19 @@ def get_cfn_attribute(self, attribute_name):
 
     @staticmethod
     def get_deploy_templates():
+        # TODO: validate again
```

Comment:
We'd assume the tests cover it :raised_eyebrow: 

There are a few templates used in our test suite that cover the resource but they seem to have been broken before and not actively part of our current integration tests

- `tests/integration/templates/template30.yaml` (marked as **skip**)
- `tests/integration/templates/template28.yaml` (has been marked as **xfail** though)
- `tests/integration/templates/valid_template.json` (only has a validate_template call, **not deployed**)

So since this might not actually have worked before this change, I'd leave the TODO in there for now in case this pops up somewhere and we need to fix it.